### PR TITLE
Index ORCID ids in Paper objects

### PR DESCRIPTION
**Pull request by wetneb**
This changes the search index so that ORCID ids are directly indexed there.

We currently only index researcher ids (we don't have researchers for every ORCID profile, hence the discrepancy).

Tests are still needed and `link_existing_papers` needs to be hooked to the rest of the code for #659.

### CODE REVIEW

**Comment by wetneb** [comment]
This is now ready. After deploying this, we will need to recompute the ES index entirely.

File: backend/orcid.py [hunk 1]
```diff
@@ -146,6 +149,35 @@ def warn_user_of_ignored_papers(self, ignored_papers):
...
+                new_authors = paper.authors
+                for author in new_authors:
+                    if author.orcid == researcher.orcid:
+                        author.researcher_id = researcher.id
+                paper.authors_list = [author.serialize() for author in new_authors]
+                papers_to_update.append(paper)
+                paper.update_index()
+        
+        if papers_to_update:
+            if django.VERSION[0] > 2 or (django.VERSION[0] == 2 and django.VERSION[1] >= 2):
```
Comment (Phyks) [inline backend/orcid.py]: Is there a main reason to support multiple Django versions?

**Comment by Phyks** [comment]
Eager to see this landing in production, and hoping that both #730 and related will be fixed :)

File: backend/orcid.py (hunk 1 above)
Comment (wetneb) [reply backend/orcid.py]: Yeah let's just require 2.2, this `bulk_update` thing is going to be useful at many other places I think

**Pull request merged**